src/software/ai/evaluation/deflect_off_enemy_target.cpp

```diff
@@ -39,7 +39,10 @@ namespace Evaluation
             {
                 if (fabs(i.position().y() - closestEdgeY) < shortestLenToEdge)
                 {
-                    enemyClosestToEdge.updateState(i);
+                    enemyClosestToEdge.updateCurrentState(
```

Comment:
nit, optional; both `enemyClosestToEdge` and `i` violate our naming conventions, `enemyClosestToEdge` should be `enemy_closest_to_edge` and `i` should be `enemy_robot` (or really anything more descriptive then `i`). Out-of-scope, so feel free to ignore, but would be good to fix them.

Reply:
done